app/components/modified-image.cjsx

```diff
@@ -0,0 +1,34 @@
+React = require 'react'
+
+FILTERS = {
+    invert: "url('#svg-invert-filter')"
+}
+
+INVERT =
+  '<svg style="position: fixed; right: 100%; top: 100%; visibility: hidden;">
+    <defs>
+      <filter id="svg-invert-filter">
+        <feComponentTransfer>
+          <feFuncR type="table" tableValues="1 0"/>
+          <feFuncG type="table" tableValues="1 0"/>
+          <feFuncB type="table" tableValues="1 0"/>
+        </feComponentTransfer>
+      </filter>
+    </defs>
+  </svg>'
+
+module.exports = React.createClass
+  displayName: 'ModifiedImage'
+
+  componentWillMount: ->
+    @setElements()
```

Comment:
You should do this once, outside the component lifecycle. As written, it'll add a new `#svg-invert-filter` every time one of these mounts.